src/licenses.ts

```diff
@@ -24,11 +24,22 @@ export async function getInvalidLicenseChanges(
   licenses: {
     allow?: string[]
     deny?: string[]
+    exception?: Record<string, string[]>
   }
 ): Promise<InvalidLicenseChanges> {
-  const {allow, deny} = licenses
+  const {allow, deny, exception} = licenses
```

Comment:
Nitpick: can we use something more explicit than `exception`? The naming is precise and good but it could be confusing for a reader to find that `exception` is not actually related to JS exceptions.

Reply:
Renamed ti to `licenseExclusions` which hopefully is a bit more clear. [9ad7edb](https://github.com/actions/dependency-review-action/pull/423/commits/9ad7edb033732c61e69c85d649c21e612fa9ad57)